.drone.yml

```diff
@@ -57,6 +57,11 @@ steps:
       - cargo workspaces exec cargo check --no-default-features
       - cargo workspaces exec cargo check --all-features
 
+  - name: lemmy_api_common doesnt depend on diesel
+    image: rust:1.64-buster
+    commands:
+      - "! cargo tree -p lemmy_api_common --no-default-features -i diesel"
```

Comment:
The exclamation point isn't necessary right?

Reply:
Its necessary because we want to ensure that the command returns an error via exit code. This means the diesel dependency was not found, which is the goal in this case. Crates which pull in lemmy_api_common to access api structs (eg lemmybb) shouldnt have to compile diesel.